crates/ide/src/doc_links.rs

```diff
@@ -362,14 +362,15 @@ fn map_links<'e>(
     // holds the origin link target on start event and the rewritten one on end event
     let mut end_link_target: Option<CowStr> = None;
     // normally link's type is determined by the type of link tag in the end event,
-    // however in same cases we want to change the link type, for example,
+    // however in some cases we want to change the link type, for example,
     // `Shortcut` type doesn't make sense for url links
```

Comment:
This comment actually confuses me, the code below makes no mention of Shortcut? I wonder if this is outdated?

Reply:
Ah I see, the comment is phrased a bit confusingly I guess.

Reply:
Oh thanks for the fix, I missed the the autolink case in the pr before.

>  the code below makes no mention of Shortcut

I meant to say the `Shortcut` types coming from original start/end events, which are deconstructed to `link_type` in the following code. Let's make the comment more clearer?

Reply:
Agreed, will do these later today when I'm home